tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/AspNetCore5IastTests.cs

```diff
@@ -46,12 +35,23 @@ public async Task TestIastWeakHashingRequestSampling()
             await TryStartApp();
             await TestWeakHashing(filename, Fixture.Agent);
 
-            filename = "Iast.WeakHashing.AspNetCore5.IastDisabled";
+            filename = "Iast.WeakHashing.AspNetCore5.IastDisabledFlag";
```

Comment:
Shouldn't you have removed previous snapshots in this PR?

Reply:
We still use that snapshot in another test: TestIastWeakHashingRequest when iast is totally disabled.